packages/frontend2/src/app/(new)/(other)/scaling/finality/_components/table/grayed-out.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+export function GrayedOut(props: {
+  children: React.ReactNode
+  grayOut: boolean
```

Comment:
Why `GrayedOut` component has `grayOut` field. I suppose it's copied from old FE, but anyway I suggest to change the naming or just remove this prop.